### DISCUSSION TITLE
QTable select by a single column value, rather than the whole row object

### DISCUSSION THREAD

**Discussion by kurtisas-k** [quasarframework/quasar#10155] [General - Components / Directives / etc] [answered]
https://codepen.io/kstaplesking/pen/ZEKXOLJ?editors=101

I was wondering how would I modify the existing code so that instead of the selected array containing an Object with each of the row fields in it, I'd instead just get the value of a single column? 

So selected would look something like:

["Ice cream sandwich","Eclair"]

instead of the current output of 

 Selected: [{"name":"Ice cream sandwich","calories":237,"fat":9,"carbs":37,"protein":4.3,"sodium":129,"calcium":"8%","iron":"1%"},{"name":"Eclair","calories":262,"fat":16,"carbs":23,"protein":6,"sodium":337,"calcium":"6%","iron":"7%"}] 

Thanks in advance for any support,

**Comment by MilosPaunovic** [comment, marked as answer]
You could always do something like this:

```js
computed: {
  names() {
    return this.selected.map(value => value.name)
  }
}
```

So the `names` will return `name` prop from selected values array.

**Reply by kurtisas-k** [reply]
That's perfect. Took me a minute to figure out how I'd use it. Thanks!
